clients/java/signalr/src/test/java/WebSocketTransportTest.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+import org.junit.Test;
+
+import java.net.URISyntaxException;
+
+import static org.junit.Assert.*;
+
+public class WebSocketTransportTest {
+
+    @Test
+    public void checkThatWsIsUnchanged() throws URISyntaxException {
```

Comment:
Looks like junit has parameterized tests. It's parameterized by class instead of by method but that could work fine here (no reason we can't use a nested class like `WebSocketTransportTests.UrlTests`)

https://github.com/junit-team/junit4/wiki/parameterized-tests

Reply:
Cool, I'll look into this